app/models/concerns/web_resource.rb

```diff
@@ -6,6 +6,26 @@ module WebResource
   extend ActiveSupport::Concern
 
   def get_json(location, limit = 10)
+    result = get_contents(location, limit)
+    if result
+      JSON.parse(result)
+    else
+      false
+    end
+  end
+
+  def get_php_serialized_data(location, limit = 10)
+    result = get_contents(location, limit)
+    if result
+      response = result.dup.force_encoding('utf-8')
+      require 'php_serialization/unserializer'
+      return PhpSerialization::Unserializer.new.run(response)
+    else
+      return []
+    end
+  end
+
+  def get_contents(location, limit = 10)
```

Comment:
__[reek]__
* [DataClump] takes parameters [limit, location] to 3 methods [:ref](https://github.com/troessner/reek/blob/master/docs/Data-Clump.md)
* [TooManyStatements] has approx 15 statements [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md)